packages/adapter-chiselstrike/tests/basic.test.ts

```diff
@@ -0,0 +1,37 @@
+import { runBasicTests } from "@next-auth/adapter-test"
+import { ChiselStrikeAdapter, ChiselStrikeAuthFetcher } from "../src"
+
+const fetcher = new ChiselStrikeAuthFetcher('http://localhost:8080', '1234')
+const adapter = ChiselStrikeAdapter(fetcher)
+
+runBasicTests(
+    {
+        adapter,
+        db: {
+            connect: async () => {
+                await fetcher.deleteEverything()
+            },
+            session: async (sessionToken: string) => {
+                const s = await fetcher.filter(fetcher.sessions(`?.sessionToken=${sessionToken}`))
+                return s ? { ...s, expires: new Date(s.expires) } : null
+            },
+            user: async (id: string) => {
+                return await adapter.getUser(id)
```

Comment:
should not test the user retrieval with the adapter itself

Reply:
Done.